pkg/server/sandbox_run_unix.go

```diff
@@ -108,6 +108,12 @@ func (c *criService) sandboxContainerSpec(id string, config *runtime.PodSandboxC
 			Type:        "bind",
 			Options:     []string{"rbind", "ro"},
 		},
```

Comment:
Can you add a comment about why is this needed?

I already forgot why we mounted dev shim, so it would be better to put a comment here.

Reply:
done, PTAL 😁